src/ucp/core/ucp_worker.c

```diff
@@ -1816,6 +1896,8 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
     *worker_p = worker;
     return UCS_OK;
 
+err_destroy_mpools:
+    ucp_worker_destroy_mpools(worker, 0);
```

Comment:
why 0 ?

Reply:
by analogy with original error handling code in ucp_worker_init_mpools. change to 1?

Reply:
ok keep as is